versions.tf

```
@@ -1,3 +1,3 @@
 terraform {
-  required_version = "~> 0.12.0"
+  required_version = "~> 0.13.0"
```

We want to support both `0.12` and `0.13` for some time.

```suggestion
  required_version = ">= 0.12.0, < 0.14.0"
```